res/controllers/Reloop-Jockey-3-ME-scripts.js

```diff
@@ -0,0 +1,546 @@
+			fxSelectKnobPushBreak: {
+				for (var r = 0, s = 1; r < Jockey3ME.fxSelectKnobPushIterator.length; r++) {
+					if ((r % 4) == 0 && r != 0) {
+						++s;
+					}
+					var variableSvalue = 0;
+					if (s == 2) {
+						variableSvalue = 4;
+					} else if (s == 3) {
+						variableSvalue = 8;
+					}
+					if ((r - variableSvalue) == (currentDeck-1)) {
+						continue;
+					}
+					if (Jockey3ME.fxSelectKnobPushIterator[r]) {
+						engine.beginTimer(400,"midi.sendShortMsg(" + status + "," + control + ",0x7F)",1);
+						engine.beginTimer(600,"midi.sendShortMsg(" + status + "," + control + ",0x00)",1);
+						break fxSelectKnobPushBreak;
+					}
+				}
+				var currentDeckKnob = currentDeck;
+				if (Jockey3ME.fxSelectKnobPush[currentDeck-1] == 2) {
+					currentDeckKnob += 4;
+				} else if (Jockey3ME.fxSelectKnobPush[currentDeck-1] == 3) {
+					currentDeckKnob += 8;
+				}
+				if (Jockey3ME.fxSelectKnobPushIterator[currentDeckKnob-1] == 0) {
+					Jockey3ME.fxSelectKnobParamLinkInverseChose = 0;
+					Jockey3ME.fxSelectKnobParamLinkChose = 0;
+					Jockey3ME.fxSelectKnobParamChose = 0;
+				}
+				++Jockey3ME.fxSelectKnobPushIterator[currentDeckKnob-1];
+				Jockey3ME.fxSelectKnobPushLedTemp = 0;
+				if (Jockey3ME.fxSelectKnobPushIterator[currentDeckKnob-1] > 3) {
+					Jockey3ME.fxSelectKnobPushIterator[currentDeckKnob-1] = 0;
+					if (engine.getValue("[EffectRack1_EffectUnit" + currentDeck + "_Effect" + Jockey3ME.fxSelectKnobPush[currentDeck-1] + "]","parameter" + Jockey3ME.fxSelectKnobParamChose + "_loaded")) {
+						engine.setValue("[EffectRack1_EffectUnit" + currentDeck + "_Effect" + Jockey3ME.fxSelectKnobPush[currentDeck-1] + "]","parameter" + Jockey3ME.fxSelectKnobParamChose + "_link_type",Jockey3ME.fxSelectKnobParamLinkChose);
+						engine.setValue("[EffectRack1_EffectUnit" + currentDeck + "_Effect" + Jockey3ME.fxSelectKnobPush[currentDeck-1] + "]","parameter" + Jockey3ME.fxSelectKnobParamChose + "_link_inverse",Jockey3ME.fxSelectKnobParamLinkInverseChose);
+						engine.beginTimer(400,"midi.sendShortMsg(" + status + "," + control + ",0x7F)",1);
+						engine.beginTimer(600,"midi.sendShortMsg(" + status + "," + control + ",0x00)",1);
+						engine.beginTimer(800,"midi.sendShortMsg(" + status + "," + control + ",0x7F)",1);
+						engine.beginTimer(1000,"midi.sendShortMsg(" + status + "," + control + ",0x00)",1);
+					}
+				}
+			}
+			Jockey3ME.fxSelectKnobPush[currentDeck-1] = 0;
+		} else {
+			Jockey3ME.fxSelectKnobPush[currentDeck-1] = 0;
+			engine.beginTimer(400,"midi.sendShortMsg(" + status + "," + control + ",0x7F)",1);
+			engine.beginTimer(600,"midi.sendShortMsg(" + status + "," + control + ",0x00)",1);
+		}
+		midi.sendShortMsg(status,control,0x7F);
+		engine.beginTimer(200,"midi.sendShortMsg(" + status + "," + control + ",0x00)",1);
+	}
+}
+
+Jockey3ME.effectSuperKnob = function (channel, control, value, status, group) {
```

Comment:
Add a comment here to make it clear that this is one of the modes of the jog wheel.